notebooks/examples/python/Ship2ShipTransfers/01. Data Ingestion.py

```diff
@@ -0,0 +1,90 @@
+# Databricks notebook source
+# MAGIC %md ## Data Ingestion
+
+# COMMAND ----------
+
+from pyspark.sql.functions import *
+import mosaic as mos
+
+spark.conf.set("spark.databricks.labs.mosaic.geometry.api", "ESRI")
+spark.conf.set("spark.databricks.labs.mosaic.index.system", "H3")
+mos.enable_mosaic(spark, dbutils)
+
+# COMMAND ----------
+
+# MAGIC %md
+# MAGIC
+# MAGIC We begin with loading from a table. Here we use captured `AIS` data.
+# MAGIC
+# MAGIC - MMSI: unique 9-digit identification code of the ship - numeric
+# MAGIC - VesselName: name of the ship - string
+# MAGIC - CallSign: unique callsign of the ship - string
+# MAGIC - timestamp: timestamp of the AIS message - datetime
+# MAGIC - LAT: latitude of the ship (in degree: [-90 ; 90], negative value represents South, 91 indicates ‘not available’) - numeric
+# MAGIC - LON: longitude of the ship (in degree: [-180 ; 180], negative value represents West, 181 indicates ‘not available’) - numeric
+# MAGIC - SOG: speed over ground, in knots - numeric
+# MAGIC - Status: status of the ship - string
+
+# COMMAND ----------
+
+cargos = spark.read.table("esg.cargos")
+display(cargos)
+
+# COMMAND ----------
+
+# MAGIC %md ## Data Transformation
+
+# COMMAND ----------
+
+# MAGIC %md
+# MAGIC We can convert the lat/lon from this table to a geometric representation. For illustrative purposes we opt for WKT representation. When storing the data later, we will opt for the more optimal WKB representation.
```

Comment:
can we use lon/lat everywhere? this way we always stick to the preferred way of specifying position.